### PR TITLE
librdkafka: install 9.1

### PR DESCRIPTION
**Pull request by mikeklein13**
Use ppa to install librdkafka 9.1

### CODE REVIEW

**Comment by bossjones** [comment]
simple change ... but needs a test. I'd check to ensure the sources file gets created correctly. should be located in `/etc/apt/sources.d` something like that. File contents should look like:

```
deb http://ppa.launchpad.net/guessi/librdkafka/ubuntu xenial main 
deb-src http://ppa.launchpad.net/guessi/librdkafka/ubuntu xenial main 
```

Also, I'm assuming alpine is already installing the latest version of libkafka and thats why you didn't need a modification to it?

**Comment by bryanlatten** [comment]
@mikeklein13 agreed with @bossjones, would love to see a version test if possible.

**Comment by bossjones** [comment]
👍

File: container/root/tests/php-fpm/beta.goss.yaml [hunk 1]
```diff
@@ -44,6 +44,10 @@ package:
     installed: true
   php7.1-zip:
     installed: true
+  librdkafka-dev:
+    installed: true
+    versions: 
+      - "0.9.1-1ppa1~xenial"
```
Comment (bryanlatten) [inline container/root/tests/php-fpm/beta.goss.yaml]: if this version gets updated, its going to break any further builds. Can you do a substring to just look for `0.9.1` instead?

**Review by bryanlatten — CHANGES_REQUESTED** [review]

**Review by bryanlatten — APPROVED** [review]

**Pull request merged**